test/blackbox-tests/test-cases/mdx-stanza/mld-files.t/run.t

```diff
@@ -0,0 +1,40 @@
+The project has an `.mld` file that needs to be fixed. At first, we determine
+that that file is not being picked up:
+
+  $ cat > dune <<EOF
+  > (mdx)
+  > EOF
+  $ dune runtest
+  File "needs-fixes.md", line 1, characters 0-0:
+  Error: Files _build/default/needs-fixes.md and
+  _build/default/.mdx/needs-fixes.md.corrected differ.
+  [1]
+
```

Comment:
there's an error at the moment in CI, the following line should be updated with the new bound:

https://github.com/ocaml/dune/blob/a04e8381cd541c137afc2f8ac31da7599d917ea6/Makefile#L18

Reply:
Also when I opened the PR MDX 2.3.0 wasn't merged to opam-repository yet so it probably didn't pick up the right version. It's merged since and I updated this line :)